src/api-reference/direct-connects/hotel-service-2/Availability.markdown

```diff
@@ -351,8 +351,9 @@ For a description of the relationship between the `RoomID` and `RatePlanID` refe
 
 |Name|Type|Description|
 |--------------------|-----------|-------------|
+|`NoCancelInd`|`boolean`| If true, the reservation may not be cancelled once the cancellation deadline has expired. False or missing flag will be treated as rate being not cancellable.|
```

Comment:
Let's use "cannot" instead of "may not" to be more assertive

Reply:
this was exact wording from OTA `may`. Changed to cannot for now